airflow-core.yaml

```diff
@@ -0,0 +1,193 @@
+package:
+  name: airflow-core
+  version: 2.10.4
+  epoch: 0
+  description: Minimal Airflow core - Platform to programmatically author, schedule, and monitor workflows
+  dependencies:
+    runtime:
+      - bash
+      - busybox
+      - coreutils
+      - curl
+      - cyrus-sasl
+      - cyrus-sasl-dev
```

Comment:
Same for these other dev dependencies

Reply:
Thanks for taking a look at them, veried and confirmed that we can proceed without dev, apologies for not taking a deeper look earlier with apk search